.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: false
 language: java
 jdk:
-- oraclejdk8
+- openjdk8
```

Comment:
why don't openjdk11

Reply:
oh this pull request is not about java11 upgrade. its for the center_name fix only. i will create another PR for that.